src/app/cases/page.tsx

```diff
@@ -85,162 +57,78 @@ export default function Page() {
       caseData
         .filter(
           c =>
-            caseFilters.remote.size === 0 ||
-            (caseFilters.remote.has('Remote') && c.is_remote) ||
-            (caseFilters.remote.has('In Person') && !c.is_remote),
+            remoteFilters.size === 0 ||
+            (remoteFilters.has('Remote') && c.is_remote) ||
+            (remoteFilters.has('In Person') && !c.is_remote),
```

Comment:
In the ProfileMatch pr, I've changed the Listing Details + ListingCard to display "Not Available" if c.is_remote is null, rather than defaulting to "In Person." This filter is still defaulting null values of `is_remote` to In Person. Do we want to change this to match the handling of `is_remote` elsewhere?

Reply:
good point! let's discuss this edge case

Reply:
from worksesh: we'll include listings with null fields always